CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -556,6 +560,12 @@ protected virtual void Dispose(bool isDisposing)
                     }
 
                     PresentationSource.RemoveSourceChangedHandler(this, PresentationSourceChangedHandler);
+                    // Release window event listeners if SourceChanged event wasn't received before disposal
+                    if (sourceWindow != null)
```

Comment:
Should this be set to `null` here as well?

Reply:
Should I squash these commits?